helmfile.d/README.md

```diff
@@ -0,0 +1,59 @@
+# Helmfile
+
+Compliant Kubernetes Apps is driven by [`helmfile`](https://github.com/helmfile/helmfile).
+
+The structure is as follows:
+
+- `bases/` - contains the environment definition and some common values
+- `charts/` - contains internally developed charts grouped per stack
+- `upstream/` - contains externally developed charts grouped per source
+- `stacks/` - contains release templates for all applications grouped per stack
+- `values/` - contains values templates for all applications grouped per stack
+- `state.yaml` - the state file that pulls in everything and defines releases
+
+## Usage
+
+While `helmfile` can be invoked via `./bin/ck8s ops helmfile <sc|wc>` it can also be invoked directly from the root of the repository.
+Note that you must then point `KUBECONFIG` to the correct cluster and invoke it as `helmfile -e service_cluster|workload_cluster`.
```

Comment:
I would add an important note here saying that there is no config version validation when using plain helmfile so one needs to be careful!